integrations/access/servicenow/client.go

```diff
@@ -223,6 +223,24 @@ func (snc *Client) GetOnCall(ctx context.Context, rotaID string) ([]string, erro
 	return emails, nil
 }
 
+// CheckHealth pings servicenow to check if it is reachable.
+func (snc *Client) CheckHealth(ctx context.Context) error {
+	resp, err := snc.client.NewRequest().
+		SetContext(ctx).
+		SetQueryParams(map[string]string{
+			"sysparm_limit": "1",
+		}).
+		Get("/api/now/table/incident")
```

Comment:
The error when that table isn't present would be included in the errwrapper section. I could include the requirement for the table in the docs?

Reply:
Do you know what the error message would look like?
If it's clear enough, than it's ok to leave as is
For the docs, I think it would be worth it to add a note (I guess that's another PR)

Also, I recall we have a plugin status field
Should this kind of error trigger a change in the status field to indicate to the user that this is not working (and why/how to fix it)
No need to do this now, just something that I think we should consider

Reply:
So do you mean using the plugin status field for errors like this where they can't be retried? Would that be  clearer than just logging those cases?

Reply:
Yeah, I think so
Assuming this is going to be run first in cloud hosted envs, customers won't have access to logs

Reply:
That sounds good to me then. Could throw that in with the PR that will be adding the auto approval flow?

Reply:
Sure :+1: 
Or a PR just to handle that scenario